### PR TITLE
[2.7] Fix docblocks about callables

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <service id="debug.debug_handlers_listener" class="%debug.debug_handlers_listener.class%">
             <tag name="kernel.event_subscriber" />
             <tag name="monolog.logger" channel="php" />
-            <argument /><!-- Exception handler -->
+            <argument>null</argument><!-- Exception handler -->
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.xml]: default is null, no?

Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.xml]: I get an empty string instead...

Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.xml]: Yes, it is handled this way in [XmlUtils:phpize()](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Config/Util/XmlUtils.php#L184-L217).

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

File: src/Symfony/Component/Console/Question/Question.php [hunk 2]
```diff
@@ -217,7 +217,7 @@ public function getMaxAttempts()
      *
      * The normalizer can be a callable (a string), a closure or a class implementing __invoke.
```
Comment (stof) [inline src/Symfony/Component/Console/Question/Question.php]: this sentence is weird too

Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Question/Question.php]: removed

**Pull request merged**